hw03_babayemie.py

```diff
@@ -0,0 +1,164 @@
+#################################################################################
+# Author: Elijah Babayemi
+# Username: babayemie
+#
+# Assignment:HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: Continue practicing creating and using functions, more practice on using the turtle library, learn about how computers represent colors, and learn about source control and Git.
+#
+# Google Doc Link: https://docs.google.com/document/d/1X9d90s3RxZa3LG8GFm7Y8XtmVSOv1ai_OpNtu0xGtpk/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
+import turtle
+
+
+
+
+def make_house():
+   """
+  Draws the main body of the house
+   """
+   turtle.penup()
```

Comment:
Naming the turtle `turtle` is a dangerous practice, because it can overwrite the whole turtle library with a turtle object, resulting in a runtime error. I always try to name objects uniquely to avoid collisions with libraries and other variables.